Iss1 create poem page

tynanook opened the pull request:
@yuens1002 @JBigs206 
This is the branch for iss1, creating a poem page. [iss1-create-poem-page](https://github.com/suwebdev/proud-dew/issues/1).
I do not know what the .idea/....xml file is.
The page is consistent with our website design. I arbitrarily populated the page with a Robert Frost poem. The reviews and the review button at the bottom section of the page is related to issue 2 (and to a lesser degree, issue 3).

Commit:
added empty poem.html page

tynanook committed:
added poem.html page. made modification to pages.html to show functionality. Poem.html has reviews listed at bottom of page, as well as button to add new reviews. this must work with review form for functionality, which is iss2.

tynanook committed:
removed poems.html from my file as it is not part of my branch.

tynanook committed:
removed blank spaces in poem.html

tynanook commented:
I added @yuens1002 and @JBigs206 as assignees, for review as they are familiar with the navigation of the website, which this page will is dependent upon.

tynanook commented:
This pull request is waiting on comments for at least 2 members of the team before it can be merged. @yuens1002 @JBigs206 @wibblywink

wibblywink commented on poem.html:
```diff
@@ -0,0 +1,208 @@
+            I took the one less traveled by,<br>
+            And that has made all the difference.<br>
+        </p>
+
```
Add a comment here so it's clear that we change from the poem section to the review. I suggest a 'Reader reviews' comment in a new line.

yuens1002 commented:
the page is broken - no style. pls first merge from one of my branches when you're in your branch.  do, git checkout "your branch name", then git merge "one of my remote branches".  you'll pull all the current site markup (pages) and css which you can use to apply styles to your poem.html file.  you can model your poem.html page off of the index.html to get all the styles to show up.  try to avoid modification of files you didn't create, then commit/push all your changes back to remote.  when merging, all your commits can be collapsed into one single merge.

is the xml file really necessary for the site, is there a reason why it's included in this issue?

tynanook committed:
fetch/merge'd from master

tynanook committed:
added poem.html, same as before w/no changes, just added to fetch'd master

tynanook commented:
@yuens1002 @wibblywink @JBigs206 
I merged the master pages into this branch to allow easier viewing of the 'poem.html' page. 
I modified the 'poems.html' to allow functional viewing of the 'poem.html' page. To view the 'poem.html' page:  from the homepage, click on the 'poems.html' --> 'titles' in the navbar menu, then either click on 'Aaron' or 'Abacus'

There review section has a title and comments added in the file.
I removed unnecessary comments. 
I'm not sure where the .xml file originated, but it is now deleted.

tynanook committed:
added comment for review section

tynanook committed:
removed unnecessary comment

JBigs206 commented:
@tynanook Looks good. I will link the poems in my dropdown menu to your poem.html page.

yuens1002 commented on poem.html:
```diff
@@ -0,0 +1,207 @@
+            <hr>
+              <p class="poem-pg-author-bio">Robert Frost was born in San Francisco, California, to journalist William Prescott Frost, Jr., and Isabelle Moodie. His mother was a Scottish immigrant, and his father descended from Nicholas Frost of Tiverton, Devon, England, who had sailed to New Hampshire in 1634 on the Wolfrana.</p>
+        </div>
+        <div class="col-md-8 "poem-pg-poem-text">
```
take out the extra ", should be

`<div class="col-md-8 poem-pg-poem-text">`

yuens1002 commented on poems.html:
```diff
@@ -109,10 +109,10 @@
           <h2 class="visible-md-block visible-lg-block">Title<br><br></h2>
 				
-          <h2>A - C</h2><p><a href="#">Aaron</a>
+          <h2>A - C</h2><p><a href="./poem.html">Aaron</a>
```
`<a href="poem.html">Aaron</a>`

this is how poem.html should be linked, b/c all of our html files sit at the same root level.

yuens1002 commented on poems.html:
```diff
@@ -109,10 +109,10 @@
 <br>BY GEORGE HERBERT
 <br>Holiness on the head,</p>
-<p><a href="#">Abacus</a>
+<p><a href="./poem.html">Abacus</a>
```
the same goes here

yuens1002 commented:
i would leave out workspace.xml on your next commit the file don't seemed to be linked or used.  would be nice to include the side nav in your layout for consistency, and to allow users to navigate back or to another section within poems.  i left inline comments regarding to your poem.html.  great work! pls merge asap.
